From: KevinDuan <duankaiwen@kylinos.cn>
Date: Fri, 14 Aug 2026 08:30:58 +0000
Subject: Translated using Weblate (Italian)

Currently translated at 97.7% (43 of 44 strings)

Translation: openkylin-huanghe/search-ukcc-plugin
Translate-URL: http://weblate.openkylin.top/projects/openkylin-huanghe/search-ukcc-plugin/it/
---
 search-ukcc-plugin/translations/it.ts | 86 +++++++++++++++++------------------
 1 file changed, 43 insertions(+), 43 deletions(-)

diff --git a/search-ukcc-plugin/translations/it.ts b/search-ukcc-plugin/translations/it.ts
index 1bd3da3..1e03484 100644
--- a/search-ukcc-plugin/translations/it.ts
+++ b/search-ukcc-plugin/translations/it.ts
@@ -7,13 +7,13 @@
         <location filename="../search.cpp" line="46"/>
         <location filename="../search.cpp" line="285"/>
         <source>Search</source>
-        <translation type="unfinished"></translation>
+        <translation>Ricerca globale</translation>
         <extra-contents_path>/Search/Search</extra-contents_path>
     </message>
     <message>
         <location filename="../search.cpp" line="302"/>
         <source>Create index</source>
-        <translation type="unfinished"></translation>
+        <translation>Crea indice</translation>
         <extra-contents_path>/Search/Create index</extra-contents_path>
     </message>
     <message>
@@ -23,18 +23,18 @@
     <message>
         <location filename="../search.cpp" line="290"/>
         <source>Default web searching engine</source>
-        <translation type="unfinished"></translation>
+        <translation>Motore di ricerca Internet predefinito</translation>
         <extra-contents_path>/Search/Default web searching engine</extra-contents_path>
     </message>
     <message>
         <location filename="../search.cpp" line="294"/>
         <source>baidu</source>
-        <translation type="unfinished"></translation>
+        <translation>Baidu</translation>
     </message>
     <message>
         <location filename="../search.cpp" line="295"/>
         <source>sougou</source>
-        <translation type="unfinished"></translation>
+        <translation>Sogou</translation>
     </message>
     <message>
         <location filename="../search.cpp" line="296"/>
@@ -44,29 +44,29 @@
     <message>
         <location filename="../search.cpp" line="449"/>
         <source>Create AI index</source>
-        <translation type="unfinished"></translation>
+        <translation>Indicizzazione AI</translation>
         <extra-contents_path>/Search/Create AI index</extra-contents_path>
     </message>
     <message>
         <location filename="../search.cpp" line="453"/>
         <source>AI model is ready</source>
-        <translation type="unfinished"></translation>
+        <translation>Modello configurato</translation>
     </message>
     <message>
         <location filename="../search.cpp" line="455"/>
         <source>AI model is not ready</source>
-        <translation type="unfinished"></translation>
+        <translation>Modello non configurato</translation>
     </message>
     <message>
         <location filename="../search.cpp" line="502"/>
         <source>Block Folders</source>
-        <translation type="unfinished"></translation>
+        <translation>Cartelle escluse</translation>
         <extra-contents_path>/Search/Block Folders</extra-contents_path>
     </message>
     <message>
         <location filename="../search.cpp" line="507"/>
         <source>Following folders will not be searched. You can set it by adding and removing folders.</source>
-        <translation type="unfinished"></translation>
+        <translation>La ricerca non controllerà le seguenti cartelle; puoi impostare le posizioni delle cartelle escluse aggiungendole o rimuovendole</translation>
     </message>
     <message>
         <source>Choose folder</source>
@@ -76,13 +76,13 @@
         <location filename="../search.cpp" line="726"/>
         <location filename="../search.cpp" line="797"/>
         <source>delete</source>
-        <translation type="unfinished"></translation>
+        <translation>Elimina</translation>
     </message>
     <message>
         <location filename="../search.cpp" line="855"/>
         <location filename="../search.cpp" line="908"/>
         <source>Directories</source>
-        <translation type="unfinished"></translation>
+        <translation>Cartella</translation>
     </message>
     <message>
         <source>File Content Search</source>
@@ -92,59 +92,59 @@
     <message>
         <location filename="../search.cpp" line="388"/>
         <source>show more results that match the keyword</source>
-        <translation type="unfinished"></translation>
+        <translation>Mostra più risultati di ricerca che corrispondono all&apos;input</translation>
     </message>
     <message>
         <location filename="../search.cpp" line="385"/>
         <source>Fuzzy Search</source>
-        <translation type="unfinished"></translation>
+        <translation>Ricerca fuzzy</translation>
     </message>
     <message>
         <location filename="../search.cpp" line="202"/>
         <source>Create file content index will increase the usage of memory and CPU, do you still want to open it?</source>
-        <translation type="unfinished"></translation>
+        <translation>Creare l&apos;indice del contenuto dei file potrebbe influire sull&apos;utilizzo di memoria e CPU, vuoi abilitarlo?</translation>
     </message>
     <message>
         <location filename="../search.cpp" line="203"/>
         <source>OK</source>
-        <translation type="unfinished"></translation>
+        <translation>Attiva</translation>
     </message>
     <message>
         <location filename="../search.cpp" line="206"/>
         <source>Devices with 8GB RAM or lower are not recommended to create AI index, as it may result in system instability (such as failure to wake from sleep or other issues).</source>
-        <translation type="unfinished"></translation>
+        <translation>Nei dispositivi con 8 GB o meno di memoria, non è consigliabile attivare l&apos;indicizzazione AI, potrebbe causare anomalie nel funzionamento del sistema (impossibilità di riattivare dal sonno, ecc.).</translation>
     </message>
     <message>
         <location filename="../search.cpp" line="208"/>
         <source>Create AI index on low-performance machines may cause the system lagging, do you still want to open it?</source>
-        <translation type="unfinished"></translation>
+        <translation>Attivare l&apos;indicizzazione AI su dispositivi a bassa prestazione potrebbe causare rallentamenti intermittenti; confermare le prestazioni del dispositivo, vuoi abilitarla?</translation>
     </message>
     <message>
         <location filename="../search.cpp" line="327"/>
         <source>Create file index</source>
-        <translation type="unfinished"></translation>
+        <translation>Indice dei file</translation>
         <extra-contents_path>/Search/Create file index</extra-contents_path>
     </message>
     <message>
         <location filename="../search.cpp" line="340"/>
         <source>Create file content index</source>
-        <translation type="unfinished"></translation>
+        <translation>Indicizzazione del contenuto dei file</translation>
         <extra-contents_path>/Search/Create file content index</extra-contents_path>
     </message>
     <message>
         <location filename="../search.cpp" line="372"/>
         <source>Precise Search</source>
-        <translation type="unfinished"></translation>
+        <translation>Ricerca precisa</translation>
     </message>
     <message>
         <location filename="../search.cpp" line="375"/>
         <source>show the results that exactly match the keyword</source>
-        <translation type="unfinished"></translation>
+        <translation>Mostra solo i risultati di ricerca identici all&apos;input</translation>
     </message>
     <message>
         <location filename="../search.cpp" line="421"/>
         <source>Index the text in pictures</source>
-        <translation type="unfinished"></translation>
+        <translation>Indicizza il testo nelle immagini</translation>
         <extra-contents_path>/Search/Use OCR to index the text in pictures</extra-contents_path>
     </message>
     <message>
@@ -155,49 +155,49 @@
     <message>
         <location filename="../search.cpp" line="469"/>
         <source>Search Folders</source>
-        <translation type="unfinished"></translation>
+        <translation>Ambito di ricerca</translation>
         <extra-contents_path>/Search/Search Folders</extra-contents_path>
     </message>
     <message>
         <location filename="../search.cpp" line="475"/>
         <source>Following folders will be searched. You can set it by adding and removing folders.</source>
-        <translation type="unfinished"></translation>
+        <translation>Il contenuto dei seguenti file apparirà nei risultati della ricerca globale</translation>
     </message>
     <message>
         <location filename="../search.cpp" line="856"/>
         <source>select blocked folder</source>
-        <translation type="unfinished"></translation>
+        <translation>Seleziona le cartelle da escludere</translation>
     </message>
     <message>
         <location filename="../search.cpp" line="857"/>
         <location filename="../search.cpp" line="910"/>
         <source>Select</source>
-        <translation type="unfinished"></translation>
+        <translation>Seleziona</translation>
     </message>
     <message>
         <location filename="../search.cpp" line="858"/>
         <location filename="../search.cpp" line="911"/>
         <source>Position: </source>
-        <translation type="unfinished"></translation>
+        <translation>Posizione </translation>
     </message>
     <message>
         <location filename="../search.cpp" line="859"/>
         <location filename="../search.cpp" line="912"/>
         <source>FileName: </source>
-        <translation type="unfinished"></translation>
+        <translation>Nome del file </translation>
     </message>
     <message>
         <location filename="../search.cpp" line="860"/>
         <location filename="../search.cpp" line="913"/>
         <source>FileType: </source>
-        <translation type="unfinished"></translation>
+        <translation>Tipo </translation>
     </message>
     <message>
         <location filename="../search.cpp" line="204"/>
         <location filename="../search.cpp" line="861"/>
         <location filename="../search.cpp" line="914"/>
         <source>Cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>Annulla</translation>
     </message>
     <message>
         <location filename="../search.cpp" line="876"/>
@@ -212,63 +212,63 @@
         <location filename="../search.cpp" line="942"/>
         <location filename="../search.cpp" line="945"/>
         <source>Warning</source>
-        <translation type="unfinished"></translation>
+        <translation>Avviso</translation>
     </message>
     <message>
         <location filename="../search.cpp" line="888"/>
         <location filename="../search.cpp" line="942"/>
         <source>Add search folder failed, hidden　path is not supported!</source>
-        <translation type="unfinished"></translation>
+        <translation>Aggiunta fallita, le directory nascoste non sono supportate!</translation>
     </message>
     <message>
         <location filename="../search.cpp" line="945"/>
         <source>Add search folder failed, permission denied!</source>
-        <translation type="unfinished"></translation>
+        <translation>Aggiunta fallita, non hai i permessi per accedere a questa directory!</translation>
     </message>
     <message>
         <location filename="../search.cpp" line="876"/>
         <source>Add blocked folder failed, its parent dir has been added!</source>
-        <translation type="unfinished"></translation>
+        <translation>Aggiunta fallita, la directory padre è già stata aggiunta!</translation>
     </message>
     <message>
         <location filename="../search.cpp" line="880"/>
         <source>Add blocked folder failed, choosen path is not exist!</source>
-        <translation type="unfinished"></translation>
+        <translation>Aggiunta fallita, il percorso da aggiungere non esiste!</translation>
     </message>
     <message>
         <location filename="../search.cpp" line="884"/>
         <source>Add blocked folder failed, it has already been blocked!</source>
-        <translation type="unfinished"></translation>
+        <translation>Aggiunta fallita, questa cartella è già stata aggiunta!</translation>
     </message>
     <message>
         <location filename="../search.cpp" line="909"/>
         <source>select search folder</source>
-        <translation type="unfinished"></translation>
+        <translation>Seleziona le cartelle da cercare</translation>
     </message>
     <message>
         <location filename="../search.cpp" line="930"/>
         <source>Add search folder failed, choosen path is not supported currently!</source>
-        <translation type="unfinished"></translation>
+        <translation>Aggiunta fallita, questa directory non è supportata al momento!</translation>
     </message>
     <message>
         <location filename="../search.cpp" line="936"/>
         <source>Add search folder failed, another path which is in the same device has been added!</source>
-        <translation type="unfinished"></translation>
+        <translation>Aggiunta fallita, la cartella si trova su un dispositivo montato duplicato, una cartella con lo stesso contenuto è già stata aggiunta!</translation>
     </message>
     <message>
         <location filename="../search.cpp" line="927"/>
         <source>Add search folder failed, choosen path or its parent dir has been added!</source>
-        <translation type="unfinished"></translation>
+        <translation>Aggiunta fallita, la directory o la sua directory padre è già stata aggiunta!</translation>
     </message>
     <message>
         <location filename="../search.cpp" line="933"/>
         <source>Add search folder failed, choosen path is in repeat mounted devices and another path which is in the same device has been added!</source>
-        <translation type="unfinished"></translation>
+        <translation>Aggiunta fallita, la cartella si trova su un dispositivo montato duplicato e la cartella di un altro punto di montaggio dello stesso dispositivo è già stata aggiunta!</translation>
     </message>
     <message>
         <location filename="../search.cpp" line="939"/>
         <source>Add search folder failed, choosen path is not exists!</source>
-        <translation type="unfinished"></translation>
+        <translation>Aggiunta fallita, il percorso da aggiungere non esiste!</translation>
     </message>
 </context>
 </TS>
